salon_creator/lib/screens/lesson_creation_screen.dart

```diff
@@ -0,0 +1,586 @@
+            _buildDeleteResourceConfirmationDialog(index);
+          },
+          icon: Icon(Icons.delete),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildSalonNameForm() {
+    return TextFieldWithLabel(
+      maxLines: 1,
+      onChanged: (_) => _updateScreenContext(),
+      controller: lessonNameController,
+      title: "レッスン名",
+      hintText: "ゴスペルピアノ",
+    );
+  }
+
+  Widget _buildCategoryForm() {
+    return TextFieldWithLabel(
+      maxLines: 1,
+      onChanged: (_) => _updateScreenContext(),
+      controller: categoriesController,
+      title: "カテゴリー",
+      hintText: "ゴスペル、ピアノ、ギター等",
+    );
+  }
+
+  Widget _buildAppBar(BuildContext context) {
+    return AppBar(
+      title: Text("レッスン作成"),
+    );
+  }
+
+  Widget _buildContentForm() {
+    return Column(
+      children: [
+        CustomLabel(
+          title: "説明文",
+        ),
+        SizedBox(
+          height: 12,
+        ),
+        _buildDescriptionTextField(),
+      ],
+    );
+  }
+
+  Widget _buildDescriptionTextField() {
+    return Container(
+      height: 216,
+      child: TextField(
+        onChanged: (_) => _updateScreenContext(),
+        controller: descriptionController,
+        textAlignVertical: TextAlignVertical.center,
+        keyboardType: TextInputType.multiline,
+        maxLength: 300,
+        maxLines: 100,
+        style: TextStyle(
+          fontSize: 16,
+        ),
+        decoration: InputDecoration(
+          contentPadding: EdgeInsets.fromLTRB(16, 8, 16, 8),
+          focusedBorder: OutlineInputBorder(
+            borderSide: BorderSide(
+              color: Colors.black,
+              width: 0.5,
+            ),
+          ),
+          enabledBorder: OutlineInputBorder(
+            borderSide: BorderSide(
+              color: Colors.black,
+              width: 0.5,
+            ),
+          ),
+          hintText: "レッスン内容",
+        ),
+      ),
+    );
+  }
+
+  Widget _buildDeleteButton(VoidCallback function) {
+    return Container(
+      width: 40,
+      height: 40,
+      margin: EdgeInsets.all(4),
+      decoration: BoxDecoration(
+        color: primaryColor,
+        shape: BoxShape.circle,
+      ),
+      child: IconButton(
+        onPressed: function,
+        icon: Icon(
+          Icons.close,
+          color: Colors.white,
+        ),
+      ),
+    );
+  }
+
+  _buildDeleteResourceConfirmationDialog(index) {
+    return showCustomDialog(
+      context: context,
+      title: Text(
+        "本当に削除しますか?",
+        textAlign: TextAlign.center,
+        style: TextStyle(
+          fontSize: 18,
+        ),
+      ),
+      content: "選択されているファイルが削除されます",
+      leftButtonText: "削除する",
+      leftFunction: () {
+        setState(() {
+          resourcesList.removeAt(index);
+        });
+
+        Navigator.pop(context);
+      },
+      rightButtonText: "取り消し",
+      rightFunction: () {
+        Navigator.of(context).pop();
+      },
+    );
+  }
+
+  removeFile() {
```

Comment:
why not make it private? Make all functions and variables private as much as possible, please.

Reply:
I just forgot to delete this function